services/webhook/webhook.go

```diff
@@ -100,13 +101,19 @@ func getPayloadBranch(p api.Payloader) string {
 	return ""
 }
 
+// SourceContext represents the source of a webhook action. Repository and/or Owner must be set.
+type SourceContext struct {
```

Comment:
`Context` is already a Golang concept. Maybe we can call it `EventSource`, because the event is triggered for a repo or a owner. `ctx SourceContext` would conflict in future if we want to introduce real Context.

And below, I think it's better be used as a pointer `src *EventSource`

Reply:
Good idea 👍